packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import SVGImage from './SVGImage'
+
+describe('SVGImage', function () {
+  it('should render an image', function () {
+    render(
+      <SVGImage
+        naturalHeight={100}
+        naturalWidth={100}
+        src='https://some.domain/image.jpg'
+        subjectID='1234'
+      />
+    )
+
+    const image = screen.getByRole('img')
+    expect(image).to.exist()
+    expect(image.getAttribute('xlink:href')).to.equal('https://some.domain/image.jpg')
+    expect(image.getAttribute('aria-label')).to.equal('Subject 1234')
```

Comment:
If you add a name to the `getByRole` query then you can probably remove this check.